torch/nn/functional.py

```diff
@@ -15,212 +15,154 @@
 from torch.autograd import Variable
 from .modules.utils import _single, _pair, _triple
 
-# Convolutions
-_ConvNd = torch._C._VariableBase._convolution
 
+conv1d = _add_docstr(torch._C._VariableBase.conv1d, r"""
+Applies a 1D convolution over an input signal composed of several input
```

Comment:
Functions which are directly bounds from C/C++ should have their type signature as the first line e.g.:

```
conv1d(input, weight, bias=None, stride=1, padding=0, dilation=1, groups=1) -> Tensor
```

Reply:
Aw, we can't autogenerate this? :)

Reply:
Please no 😂